apps/calibre/calibre_viewer.py

```diff
@@ -0,0 +1,30 @@
+from talon import Module, Context, actions
+
+# --- App definition ---
+mod = Module()
+mod.apps.calibre_viewer = """
+app: calibre
+title: /E-book viewer$/
+title: /eBook-Betrachter$/
+"""
+
+# Context matching
+ctx = Context()
+ctx.matches = """
+os: windows
+os: linux
+app: calibre_viewer
```

Comment:
I think this needs to be
```suggestion
os: windows
and app: calibre_viewer
os: linux
and app: calibre_viewer
```

But maybe I'm missing something?

Reply:
Those are equivalent.

> Each kind of requirement can be listed several times. Entries of the same kind of requirement are OR‘d together, and of different kinds are AND‘d.

See: https://talon.wiki/unofficial_talon_docs/#context-header

Reply:
Oh wow TIL thanks